src/components/nodes/pool/pool.vue

```diff
@@ -378,7 +380,7 @@ export default {
         .getElements()
         .filter(({ component }) => component && component !== this)
         .forEach(({ component }) => {
-          if (component.node.type === 'processmaker-modeler-boundary-timer-event'){
+          if (component.node.type.match(boundaryEventsPattern)){
```

Comment:
I think we can check this the same way we check it in `Modeler.vue`, using the BPMN type:
```js
component.node.definition.$type === 'bpmn:BoundaryEvent'
```
This keeps the check consistent (maybe you can pull it out into a util function) and also doesn't require regex.

Reply:
💯 on using a util function!